Task/Tasks/TSKSelectorTask.h

```diff
@@ -77,7 +93,7 @@
      May not be NULL.
  @result A newly initialized TSKSelectorTask instance with the specified name, target, and action.
  */
-- (instancetype)initWithName:(NSString *)name target:(id)target selector:(SEL)selector;
+- (instancetype)initWithName:(nullable NSString *)name target:(id)target selector:(SEL)selector;
```

Comment:
> Oh, actually, all weak properties are implicitly nullable

Are you sure? That may be a Swift 3 thing. In my environment, a `weak` objc property appears as `Type!` in swift, but if I specify it as nullable, it is `Type?`